### ISSUE TITLE
Why does this lib require babel-polyfill?

### ISSUE THREAD

**Issue by olegskl** [wavded/babel-tape-runner#15]
I'm a maintainer of [gulp-stylelint](https://github.com/olegskl/gulp-stylelint) which uses babel-tape-runner to run tests. The tests are run on Travis in Node v0.12, v4, and v5.

One day I've added `Object.assign` to my code, oblivious to the fact that it doesn't exist in Node v0.12. The tests **didn't fail**, so I published a release and soon received [a bug report](https://github.com/olegskl/gulp-stylelint/issues/11).

I've looked at the babel-tape-runner code and found [this line](https://github.com/wavded/babel-tape-runner/blob/3dd8ce22cb666c29dcfb6ab06a44dfd3f7b09f4d/bin/babel-tape-runner#L4):

``` js
require('babel-polyfill')
```

Which obviously polyfills `Object.assign`, but I'm not sure why babel-tape-runner itself needs it. There's nothing to polyfill in babel-tape-runner.

The problem is that polyfills are also applied to the code under test. Is it possible to remove them?

**Comment by wavded** [comment]
Reference: https://github.com/wavded/babel-tape-runner/issues/7

Historically, Babel 5 had it included by default in their `register`.
No way to include it in the `.babelrc` (still?) so if you wanted it, you'd have to either include it in every test file or in the first test that is "required".

See your point though, perhaps a flag to turn it off?

**Comment by olegskl** [comment]
Perhaps, instead of a flag, it would be an interesting idea to add a `--require` option similar to [what mocha does](http://mochajs.org/#usage). So we could do:

``` sh
babel-tape-runner test/*.js --require babel-polyfill
```

**Comment by wavded** [comment]
I like that.  Care to do a PR?  That would be breaking change but feels like a better API to me.

**Comment by olegskl** [comment]
After forking and looking at the code more closely I came to a conclusion that this library is not needed at all and can safely be deprecated in favor of using vanilla [tape](https://github.com/substack/tape) which already [has the require flag](https://github.com/substack/tape#preloading-modules).

My scenario calls for:

``` sh
tape -r babel-register test/*.spec.js
```

People using async/await in their tests might be interested in:

``` sh
tape -r babel-register -r babel-polyfill test/*.spec.js
```

**Comment by snuggs** [comment]
@olegskl how is this coming along? Can you submit a PR? I concur it isn't required anymore. /cc @wavded

**Comment by olegskl** [comment]
This is not required anymore. Tape can work with babel out of the box.

**Comment by wavded** [comment]
Yep, @olegskl is right.  This lib is not needed anymore.  It would be preferred to use the `tape` lib directly.
